policyengine_us/parameters/gov/states/in/tax/income/deductions/military_service/max.yaml

```diff
@@ -10,3 +11,5 @@ metadata:
       href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-2-4
     - title: 2023 IT-40 Income Tax Form Instruction Booklet, pg. 17
       href: https://forms.in.gov/Download.aspx?id=15792
+    - title: 2024 IT-40 Income Tax Form Instruction Booklet, pg. 18
```

Comment:
I see from this:
>For 2024, the deduction for active duty military compensation has been increased to 100% of compensation. See page 18 for more information.

But the statute still has the cap. Could you leave a comment above the statute reference noting that it hasn't yet been reflected in statute? If you could also find a bill that does this that'd be helpful.